### PR TITLE
User story 10

### PR DESCRIPTION
**Pull request by MatthewTLim**
# Describe the changes below:
This PR will add all functionality and test for user story 10
# Relevant user story:
User Story 10
# Specific feedback requests? Include file name and line:

# Before submitting, check the following:
- [X] Entire test suite is passing
- [99.59] SimpleCov test percentage

# Note any co-authors / anything else you'd like to add:

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -16,9 +16,14 @@ def edit
   def update
     @merchant = Merchant.find(params[:merchant_id])
     @item = Item.find(params[:id])
+    # require 'pry'; binding.pry
```
Comment (awiley33) [inline app/controllers/items_controller.rb:19]: just a heads up, you left a pry in here

File: spec/features/merchant_spec.rb [hunk 2]
```diff
@@ -97,4 +97,58 @@
...
+          visit "/merchants/#{@merchant_1.id}/items"
+
+          click_button("Enable #{@item_1.name}")
+
+          expect(page).to have_content("Enabled Items")
+          expect(page).to have_content("Disabled Items")
+
+          expect(@item_1.name).to appear_before("Disabled Items", only_text: true)
+
+          expect(@item_2.name).to_not appear_before("Disabled Items", only_text: true)
```
Comment (awiley33) [inline spec/features/merchant_spec.rb:150]: nice use of orderly testing helpers

**Review by awiley33 — APPROVED** [review]
looks good! i left some comments

**Review by dani-wilson — APPROVED** [review]
Nice job!

**Pull request merged**